aiven/resource_clickhouse_user.go

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2018-2022 Aiven, Helsinki, Finland. https://aiven.io/
+package aiven
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/aiven/aiven-go-client"
+	"github.com/aiven/terraform-provider-aiven/aiven/internal/schemautil"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+var aivenClickhouseUserSchema = map[string]*schema.Schema{
+	"project":      commonSchemaProjectReference,
+	"service_name": commonSchemaServiceNameReference,
+
+	"username": {
+		Type:        schema.TypeString,
+		Required:    true,
+		ForceNew:    true,
+		Description: complex("The actual name of the Clickhouse user.").forceNew().referenced().build(),
+	},
+	"password": {
+		Type:        schema.TypeString,
+		Sensitive:   true,
+		Computed:    true,
+		Description: "The password of the clickhouse user ( not applicable for all services ).",
```

Comment:
I think we can drop the `( not applicable .. )` since this only applies to clickhouse!